web/pages/speakers/[slug].tsx

```diff
@@ -5,27 +5,19 @@ import SectionBlock from '../../components/SectionBlock';
 import Heading from '../../components/Heading';
 import Nav from '../../components/Nav';
 import { imageUrlFor } from '../../lib/sanity';
-import { Section } from '../../types/Section';
 import TextBlock from '../../components/TextBlock';
 import { Session } from '../../types/Session';
 import Sessions from '../../components/Sessions';
 import PageContainer from '../../components/PageContainer';
 import speakerStyles from '../../pageResources/speakers/Speaker/Speaker.module.css';
+import { Person } from '../../types/Person';
 
 const QUERY = `
   {
-    "speaker": *[_id == $_id][0] {
-      _id,
-      name,
-      title,
-      bio,
-      "twitter": social.twitter,
-      photo,
+    "speaker": *[slug.current == $slug][0] {
```

Comment:
Sanity slug uniqueness is based on document type, so I think this query should specify `_type` as well. Something like this (not tested):

```suggestion
    "speaker": *[_type == 'person' && slug.current == $slug][0] {
```